core-spi-test/src/main/java/org/ehcache/internal/tier/AuthoritativeTierFlush.java

```diff
@@ -117,28 +117,4 @@ public void entryDoesNotExist() {
     assertThat(tier.flush(key, valueHolder), is(equalTo(false)));
   }
 
-  @SPITest
-  @SuppressWarnings("unchecked")
-  public void exceptionWhenValueHolderIsNotAnInstanceFromTheCachingTier() throws LegalSPITesterException {
```

Comment:
Whats the story here? I think I understand why this test isn't valid... but how did it ever make it in to the SPI test suite?  In any case a separate commit with some explanation, and maybe even an issue for it's removal might make sense - since removing an SPI test seems like a bad thing normally.